kits/sales/sales-to-cs-handoff-automation/prompts/sales-to-cs-handoff-automation_instructor-llmnode-1_user_1.md

```
@@ -0,0 +1,9 @@
+Validate the following onboarding payload.
+
+Company Name: {{triggerNode_1.output.company_name}}
+Deal Value: {{triggerNode_1.output.deal_value}}
+Sales Transcript: {{triggerNode_1.output.sales_transcript}}
+CRM Notes: {{triggerNode_1.output.crm_notes}}
+Timeline: {{triggerNode_1.output.timeline}}
+
+Return JSON with validation_status, continue_pipeline, and reason.
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)